java/src/main/java/io/chubao/exception/IOException.java

```diff
@@ -0,0 +1,10 @@
+package io.chubao.exception;
+
+
+import static io.chubao.exception.StatusCodes.CFS_STATUS_IO;
+
+public class IOException extends CfsException {
```

Comment:
Why not directly use the built-in `java.io.IOException` as IO related error exception.

Reply:
You are right, I will modify it right away.